misc/bash_completion.sh

```diff
@@ -0,0 +1,43 @@
+__worldname_list() {
+  local string output
+  string=$(luanti --worldlist name)
```

Comment:
Is there a way in Bash / ZSH completion system to execute the completed command instead of relying on the system PATH one? In this case it doesn't work if you try to complete standalone builds for exemple. (my ZSH completion script have this problem too)

It seems like PowerShell allows you to access the full command in the completion provider so you can guess the executable if you don't store the executable path in an environment variable for exemple.